.github/workflows/repo-health-job.yml

```diff
@@ -0,0 +1,103 @@
+name: Repo Health Checks
+
+on:
+  workflow_call:
+    secrets:
+      REPO_HEALTH_BOT_TOKEN: 
+        required: true
+      REPO_HEALTH_BOT_EMAIL:
+        required: true
+    inputs:
+      ORG_NAMES:
+        description: "Space separated list of organisation names to parse repos. i.e. 'openedx edx . . .' "
+        type: string
+        default: "edx openedx"
+      EDX_REPO_HEALTH_BRANCH:
+        description: "Branch of the edx-repo-health repo to check out"
+        type: string
+        default: "master"
+      ONLY_CHECK_THIS_REPOSITORY:
+        description: "If you only want to run repo health on one repository, set this to org/name of said repository."
+        type: string
+        default: ""
+      REPORT_DATE:
+        description: "The date for which repo health data is required.(format: YYYY-MM-DD)"
+        type: string
+        default: ""
+      REPO_HEALTH_GOOGLE_CREDS_FILE:
+        description: "Link to the repo health credentials file"
+        type: string
+        default: ""
+      REPO_HEALTH_OWNERSHIP_SPREADSHEET_URL:
+        description: "URL for the repo health spreadsheet to populate"
+        type: string
+        default: ""
+      REPO_HEALTH_REPOS_WORKSHEET_ID:
+        description: "Repo health repositories sheet ID"
+        type: number
+        default: 0
+
+jobs:
+  repo_health_check:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: ["3.8"]
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: setup python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+      
+      - name: Install pip-tools
+        run: python3 -m pip install pip-tools
+      
+      - name: Clone edx-repo-health repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/edx-repo-health
+          path: edx-repo-health
+          ref: ${{ inputs.EDX_REPO_HEALTH_BRANCH }}
+      
+      - name: Clone testeng-ci repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/testeng-ci
+          path: testeng-ci
+          ref: "master"
+      
+      - name: Clone repo-tools repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/repo-tools
+          path: repo_tools
+          ref: master
+      
+      - name: Clone repo-health-data repo
+        uses: actions/checkout@v3
+        with:
+          repository: edx/repo-health-data
```

Comment:
This should be a parameter, since anybody other than 2U will be storing their data in a different repo.